linear_weight: 100.0
linear_weight_opening: 1.0

angular_weight: 10.0
angular_weight_opening: 0.0

max_reach: 1.0
reach_weight: 100
reach_weight_slope: 10

obstacle_weight: 1000
obstacle_radius: 0.15
obstacle_weight_slope: 0

contact_weight: 10.0

object_weight: 10
object_tolerance: 0.01

grasp_translation_offset: [0.0, 0.0, 0.0]
grasp_orientation_offset: [0, -0.7071068, 0, 0.7071068]    # -90deg around y

upper_joint_limits: [2.8973, 1.7628, 2.8973, 0.0698, 2.8973, 3.7525, 2.8973]
lower_joint_limits: [-2.8973, -1.7628, -2.8973, -3.0718, -2.8973, -0.0175, -20.8973]
joint_limit_cost: 1000
joint_limit_slope: 100
